File: py/1255.py
n = int(input())
while n > 0:
    alphabet = [0 for i in range(26)]
    sentence = input().lower()
    
    for i in sentence:
        if i in 'abcdefghijklmnopqrstuvwxyz':
            alphabet[ord(i) - 97] += 1
            
    maximum = max(alphabet)
    mostFrequent = []
    
    for i in range(len(alphabet)):
        if alphabet[i] == maximum:
            mostFrequent.append(chr(i + 97))

    print(*mostFrequent, sep='')
    
    n -= 1

File: cpp/1255.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int n;
    cin >> n;
    getchar();

    for (n; n > 0; n--) {
        int alphabet[26] = {0}, len, max = 0, i, j;
        string sentence, result = "";
        getline(cin, sentence);
        len = sentence.length();

        for (i = 0; i < len; i++) {
            if (isalpha(sentence[i])) {
                j = (int)(tolower(sentence[i])) - 97;
                alphabet[j]++;

                if (max < alphabet[j]) {
                    max = alphabet[j];
                }
            }
        }

        for (i = 0; i < 26; i++) {
            if (alphabet[i] == max) {
                result += (char)(i + 97);
            }
        }

        cout << result << '\n';
    }

    return 0;
}